components/builder-depot/src/handlers/integrations.rs

```diff
@@ -16,34 +16,51 @@ use std::collections::HashMap;
 use iron::status::{self, Status};
 use iron::prelude::*;
 use router::Router;
-use hab_net::http::controller::*;
+use http_gateway::http::controller::*;
 use bodyparser;
 use protocol::originsrv::*;
 use protocol::net::{NetOk, ErrCode};
 use persistent;
 use DepotUtil;
-use bld_core;
+use base64;
+use bldr_core;
+use hab_core::crypto::BoxKeyPair;
 
-use super::super::server::{route_message, check_origin_access};
+use super::super::server::check_origin_access;
 
 pub fn encrypt(req: &mut Request, content: &str) -> Result<String, Status> {
     let lock = req.get::<persistent::State<DepotUtil>>().expect(
         "depot not found",
     );
     let depot = lock.read().expect("depot read lock is poisoned");
 
-    match bld_core::integrations::encrypt(&depot.config.key_dir, content) {
-        Ok(c) => Ok(c),
-        Err(_) => Err(status::InternalServerError),
-    }
+    let kp = match BoxKeyPair::get_latest_pair_for(
```

Comment:
This looks like the old code... rebase problem?

Reply:
Yep, got it! Lots of changes here caused a daily rebase of ~30 mins - Please keep an eye out for more of these!